mono/metadata/remoting.c

```diff
@@ -444,6 +449,23 @@ mono_remoting_wrapper (MonoMethod *method, gpointer *params)
 	return NULL;
 } 
 
+static MonoException *
+mono_remoting_update_exception (MonoException *exc)
+{
+	MonoClass *klass = mono_object_get_class ((MonoObject*)exc);
+
+	if (!(mono_class_get_flags (klass) & TYPE_ATTRIBUTE_SERIALIZABLE)) {
+		MonoException *ret;
```

Comment:
Does that mean we are loosing the original exception? Is that the behaviour on .NET?